plugins/woocommerce-blocks/assets/js/base/components/cart-checkout/place-order-button/index.tsx

```diff
@@ -23,14 +33,48 @@ const PlaceOrderButton = ( {
 		waitingForRedirect,
 	} = useCheckoutSubmit();
 
+	const { cartTotals } = useStoreCart();
+	const totalsCurrency = getCurrencyFromPriceResponse( cartTotals );
+
+	const buttonLabel = (
+		<div
+			// Hide this from screen readers while the checkout is processing. The text will not be removed from the
+			// DOM, it will just be hidden with CSS to maintain the button's size while the spinner appears.
+			aria-hidden={ waitingForProcessing || waitingForRedirect }
+			className={ clsx(
+				'wc-block-components-checkout-place-order-button__text',
+				{
+					'wc-block-components-checkout-place-order-button__text--visually-hidden':
+						waitingForProcessing || waitingForRedirect,
+				}
+			) }
+		>
+			{ label }
+			{ showPrice && (
+				<div className="wc-block-components-checkout-place-order-button__separator">
```

Comment:
Just curious if we tried to do this separator using CSS pseudo classes or anything to make customisation easier?

Reply:
@opr cool. If it helps, if you're able to inject the element using `:after` and its in the wrong position, [you can set the order of flex items](https://developer.mozilla.org/en-US/docs/Web/CSS/CSS_flexible_box_layout/Ordering_flex_items#the_order_property).